sdk/go/common/util/gitutil/git.go

```diff
@@ -127,6 +133,8 @@ func GetGitRemoteURL(repo *git.Repository, remoteName string) (string, error) {
 }
 
 // IsGitOriginURLGitHub returns true if the provided remoteURL is detected as GitHub.
+//
+// Deprecated: Use `strings.Contains(remoteURL, "github.com")` instead.
```

Comment:
We don't use it anywhere, and AFAICT it hasn't been used in the 5 years since it was added. Figured it was unintentionally made public, so worth marking deprecated given that it's unused.